ci/tasks/build-wheel.yml

```diff
@@ -0,0 +1,20 @@
+platform: linux
+
+inputs:
+  - name: pypi.git
+
+outputs:
+  - name: wheel
+
+run:
+  path: sh
+  args:
+  - -exc
+  - |
+    apt-get -q update
+    apt-get -yq install virtualenv
+    pip install --upgrade pip && \
+    pip --no-input install pipenv
+    # cp version/version pypi.git/.version
```

Comment:
This is not needed or? should we remove?

Reply:
Done, it's not used.